### PR TITLE
Basic restructure

### PR DESCRIPTION
**Pull request by al-duss**
Created some new general guide pages to be completed.

Made this to start a discussion on how we can restructure, but I'm thinking something like this could be a nice starting point

-ZKay

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -139,7 +139,15 @@
 
 NAVIGATION_LINKS = {
     DEFAULT_LANG: (
-
+        (
+            (
+                ("/pages/guide.md", "General Guide"),
+                ("/pages/pvt.md", "PvT"),
+                ("/pages/pvz.md", "PvZ"),
+                ("/pages/pvp.md", "PvP"),
```
Comment (StanczakDominik) [inline conf.py]:
```suggestion
                ("/guide/", "General Guide"),
                ("/pvt/", "PvT"),
                ("/pvz/", "PvZ"),
                ("/pvp/", "PvP"),
```
This is sort of non-intuitive, but it's how I had to wire lines 266-269 for the About page to show up as the main page. We could change that, of course.

File: pages/guide.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+.. description:
+.. type: text
+.. author: ZKay
+.. hidetitle: True
+.. nocomments: True
+-->
+
+## Welcome to SC2Aiur's general guide to playing Protoss
+
+### For specific matchup help, head over to [PvT](/pages/pvt.md), [PvP](/pages/pvp.md) or [PvZ](/pages/pvz.md)
```
Comment (StanczakDominik) [inline pages/guide.md]:
```suggestion
### For specific matchup help, head over to [PvT](/pvt/), [PvP](/pvp/) or [PvZ](/pvz/)
```

same as before!

File: pages/pvz.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+#### Scouting
+*TEXT ON HOW TO SCOUT*
+
+## Build Orders
+
+### Economic
+
+### Timing Attack
+[4 Gate Glaives into 8 gate glaives w/ immortals](/posts/Zest's-4-Gate-Glaives-into-8-Gate-Glaives-+-2-Immortals)
```
Comment (StanczakDominik) [inline pages/pvz.md]:
```suggestion

* [4 Gate Glaives into 8 gate glaives w/ immortals](/posts/Zest's-4-Gate-Glaives-into-8-Gate-Glaives-+-2-Immortals)
```

Let's have a list here ;)

File: pages/pvt.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+#### Scouting
+*TEXT ON HOW TO SCOUT*
+
+## Build Orders
+
+### Economic
+
+### Timing Attack
+[Parting 4-Gate Blink-Allin](/posts/PartinGs-PvT-4-Gate-Blink-Allin)
```
Comment (StanczakDominik) [inline pages/pvt.md]:
```suggestion

* [Parting 4-Gate Blink-Allin](/posts/PartinGs-PvT-4-Gate-Blink-Allin)
```

**Review by StanczakDominik — CHANGES_REQUESTED** [review]
That's a great basis! A few comments:

File: conf.py (hunk 1 above)
Comment (al-duss) [reply conf.py]: Good to know, didn't actually know how it worked :P

**Review by HappyCaterpillar — APPROVED** [review]

File: pages/pvz.md (hunk 3 above)
Comment (al-duss) [reply pages/pvz.md]: Yea, gonna need many lists in the build orders

**Review by StanczakDominik — APPROVED** [review]

**Pull request merged**